week_1/3_apr/position.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Position Practice</title>
+</head>
+
+<body style="height: 2000px;padding-top: 60px;padding-left:60px;">
+    <div style="background-color: black; height: 50px; position: fixed;top:0;right:0;left:0;color:white;">header</div>
+    <div style="background-color: black; width: 50px; position: fixed;top: 50px;;bottom:0;left:0;color:white;">sidebar
```

Comment:
@codal-kcolaco can you please check why there are 2 semicolon here `top: 50px;;`? please make sure to avoid this kind of mistakes

Reply:
@abhi-codal Yes sorry for that. Will make sure from next time.